### PR TITLE
Remove trailing slash from `platform_id`

### PR DESCRIPTION
**Pull request by knowuh**
We might *not* want to merge this PR. This change is one which was proposed, but we didn't really finalize.  But because it was top of mind, I thought I would at least drop in the PR, incase we want it.

If we do merge this, it would mean that we have to be sure to re-run the run export / import tasks on Portal and Lara staging before we QA again.

We discussed in [PT #166241318](https://www.pivotaltracker.com/story/show/166241318) remove the trailing slash from the `platform_id`.  The main advantage was that it might be able to concatenate with other strings maybe.

This PR basically adds a method to the ApplicationController `platform_id` which just returns `root_url` with the trailing slash removed.

It also adds a spec test, and updates the two controllers which publish `platform_id` values. They delegate that responsibility to `ApplicationController#platform_id`

### CODE REVIEW

**Comment by pjanik** [comment]
I think the main advantage is that it might be a bit less error-prone in the future.
We agree that other apps should never make any assumptions about `platform_id` and treat it as an opaque string. But we know that sooner or later, someone will try to parse `platform_id` out of the Portal URL. My impression is that ID without trailing space is a bit more natural. But it's just my feeling, which is pretty random, and actually, another developer can think the other way.

I guess if it's a lot of work to run these migrations, it's not that critical.

**Comment by knowuh** [comment]
@scytacki when you get a chance, have a look at 394f2fb -- I think it addresses your request.

File: app/controllers/api/v1/jwt_controller.rb [hunk 1]
```diff
@@ -95,7 +95,7 @@ def firebase
         # Firebase auth rules expect all the claims to be in a sub-object named "claims".
         # All the new properties should go there. Other apps can still read them.
         :claims => {
-          :platform_id => platform_id, # see `application_controller.rb#platform_id`
+          :platform_id => APP_CONFIG[:site_url]
```
Comment (knowuh) [inline app/controllers/api/v1/jwt_controller.rb]: We need a comma here `:platform_id => APP_CONFIG[:site_url],`

File: app/controllers/api/v1/jwt_controller.rb [hunk 2]
```diff
@@ -66,7 +66,7 @@ def firebase
...
```
Comment (knowuh) [inline app/controllers/api/v1/jwt_controller.rb]: We need a comma here `:platform_id => APP_CONFIG[:site_url],`

**Comment by knowuh** [comment]
@scytacki  I added some missing commas  in 76c42b2 . Hopefully the spec tests will pass, and then I merge if you are OK with it.

**Pull request merged**